CHANGELOG.md

```diff
@@ -1,26 +1,71 @@
-## 1.5.5 (May 1, 2020)
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Is not the main point of Keep a Changelog that changes are split per type?

Reply:
The main point of Keep a Changelog is to **keep** a changlog for humans. 😊 No one has to follow the format ideologically. Nearly all rules are recommendations, that have stood the test of time.

If the CHANGELOG.md is not up-to-date or not written at all, then it does not help anyone. Splitting them into groups could be a next step if the maintainers gives the format a 👍.

Reply:
I fully agree, I am just not sure if we should say it is based on Keep a Changelog when it does not follow one of its main recommendations.

Reply:
I think following 5 of the 7 Guiding Principles, and using rfc3339 for dates is a good start. 😊 Splitting the changes into groups requires more time in each issue and before I do that work, I would like to know if that would be accepted by the maintainers. The same goes for marking deprecations or breaking changes.